djangosaml2idp/views.py

```diff
@@ -34,31 +37,60 @@
 except AttributeError:
     raise ImproperlyConfigured("SAML_IDP_SPCONFIG not defined in settings.")
 
+user_identity = getattr(settings, 'SAML_IDP_IDENTIFYING_ATTRIBUTE', 'username')
+
+
+def build_url(*args, **kwargs):
+    get = kwargs.pop('get', {})
```

Comment:
Could `get` be renamed to something like `get_params`? I find that to be a bit clearer to understand at first sight.